pkg/cidata/cidata.TEMPLATE.d/boot/10-alpine-prep.sh

```diff
@@ -32,3 +32,6 @@ rc-service cgroups start
 # `limactl stop` tells acpid to powerdown
 rc-update add acpid
 rc-service acpid start
+
+# enable networking explicitly here for apk
+rc-service networking start
```

Comment:
cc @jandubois

Reply:
This is really a bug in `lima-init`, as it works correctly with `cloud-init`. So I think it should just be fixed in the lima-init openrc script instead.

Reply:
I've implemented the fix in https://github.com/lima-vm/alpine-lima/pull/19

Reply:
dropped